scriptmodules/emulators/retroarch.sh

```diff
@@ -56,6 +57,7 @@ function build_retroarch() {
     isPlatform "arm" && params+=(--enable-floathard)
     isPlatform "neon" && params+=(--enable-neon)
     isPlatform "x11" && params+=(--enable-vulkan)
+    isPlatform "vero4k" && params+=(--enable-dbus --enable-libusb --enable-threads --enable-mali_fbdev --enable-zlib --enable-alsa --enable-freetype --disable-x11 --disable-vulkan --with-opengles_libs='-L/opt/vero3/lib')
```

Comment:
some of these flags we currently "autodetect" - but if we are to force them, we may want to do it for all the platforms - eg enabling zlib is not vero4k specific.

Reply:
This was one of the first things I built, before I'd even considered a full blown RetroPie fork.  My build params are belt and braces from the day.
I've just reviewed the configure script, done some test builds and `./retroarch --features`.  Nearly everything is auto-detected by the build system, based on installed packages.
My params are now leaner to say the least!  How embarrassing.